src/docs/manual/index.html

```
@@ -2651,8 +2651,14 @@ <h3 id="not-enough-tests-created">Randoop does not create enough tests, or creat
 Two possible reasons for a long-running test are:
 </p>
 <ul>
-  <li>Randoop is executing a test that contains nonterminating behavior,
-    or that takes very long to execute.
+ <li>Randoop is executing a test that
+    <ul>
+      <li>contains nonterminating behavior; for example, some implementations
+      of <code>equals()</code>, <code>hashCode()</code>, etc. do not terminate
+      when a collection contains itself, or
+      <li>takes very long to execute.  For example, maybe Randoop passed a value
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </li></li> ], start tag match failed [ <li> ] on line 2659.

(tag-pair)